bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -46,18 +46,20 @@ <h2 class="c-sub-navigation-title">
 
   {% block channels %}{% endblock %}
 
-  {% call picto_card(
-    title=ftl('firefox-channel-see-something-that-isnt-working'),
+  <aside class='mzp-t-picto-center'>
+  {% call picto(
+    image_url='img/icons/warning.svg',
     heading_level=2,
-    class='bugzilla',
-    base_el='section',
-    custom_desc=True) %}
-
-  <ul>
-    <li><a class="mzp-c-cta-link" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ ftl('firefox-channel-file-a-bug-now') }}</a></li>
-    <li><a class="mzp-c-cta-link" href="https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines">{{ ftl('firefox-channel-tips-for-filing-a-bug') }}</a></li>
-  </ul>
-  {% endcall %}
+    image_width=44,
+    title=ftl('firefox-channel-see-something-that-isnt-working'),
+    body=True,
+    ) %}
+      <ul>
+        <li><a class="mzp-c-cta-link" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ ftl('firefox-channel-file-a-bug-now') }}</a></li>
+        <li><a class="mzp-c-cta-link" href="https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines">{{ ftl('firefox-channel-tips-for-filing-a-bug') }}</a></li>
```

Comment:
Not related to this PR, but I just noticed this page is a 404 (MDN just moved a bunch of Firefox docs off of MDN). I'm not sure where the new page is but we could update this link while we're in here anyway. I'll see if I can find the new address.

Reply:
Any idea where I could find the correct link?

Reply:
The original page doesn't seem to have been moved, just deleted, so I don't think there's a new URL for the same content. I think we could link to `https://support.mozilla.org/kb/contributors-guide-writing-good-bug` instead (though that page also links to the now-dead MDN page).

Reply:
Oh, odd. Shall I just also delete the sentence that is linking to the 404 page in the link you just pasted?

Reply:
That's up to whoever manages that content on SUMO.